cmds/admin_server/ui/src/search_logs/log_table/date_cell/date_cell.tsx

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import { Cell, CellProps } from 'rsuite-table';
 
+const recAccess = (rowData: any, dataKey?: string): any => {
```

Comment:
i get that you wanna be generic here but youre inside the DateCell component and are only using this func to get a Data type field
```suggestion
function resolveDatakey(rowData: any, dataKey?: string): Date {
```